### PR TITLE
ELMU #171 Inform about invalid password reset confirmation code

### LINKED ISSUE #171: Inform users about invalid/expired password reset completion codes
If a password reset completion code
- has already been used
- is invalid
- has expired

we should not simply show "404" on the complete-password-reset page, rather we want to give the user a useful message and options to proceed (like go to home page).

Test: refresh the password reset completion page

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes #171

### CODE REVIEW

File: src/components/pages/complete-password-reset.yml [hunk 1]
```diff
@@ -22,3 +22,9 @@ passwordChangedSuccessfully:
 redirectMessage:
   en: You will be redirected to the <0>logon page</0> in {seconds, plural, =1 {# second} other {# seconds}}.
   de: Sie werden in {seconds, plural, =1 {# Sekunde} other {# Sekunden}} auf die <0>Anmeldeseite</0> weitergeleitet.
+passwordChangeFailure:
+  en: The password changing code is invalid or expired.
```
Comment (laurentiu-ilici) [inline src/components/pages/complete-password-reset.yml:26]: I guess "The password reset code is invalid..." would have been better here, we keep talking about password reset (not changing) :).

File: src/components/pages/complete-password-reset.js [hunk 2]
```diff
@@ -130,13 +130,24 @@ class CompletePasswordReset extends React.Component {
       </div>
     );
 
+    const completionFailureNotice = (
+      <div className="CompletePasswordResetPage-message">
```
Comment (laurentiu-ilici) [inline src/components/pages/complete-password-reset.js:134]: Also here the naming is a bit confusing, Password reset vs changing.

File: src/components/pages/complete-password-reset.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/pages/complete-password-reset.yml:26]: Was already merged when you left the comment, but I can adjust directly on master, thanks.

File: src/components/pages/complete-password-reset.js (hunk 2 above)
Comment (adrianaluchian) [reply src/components/pages/complete-password-reset.js:134]: Now I'm confused, cause here the naming is already "reset".

Comment (laurentiu-ilici) [reply src/components/pages/complete-password-reset.js:134]: But the next line says change :)

**Pull request merged**